devices/motorola-potter/default.nix

```diff
@@ -0,0 +1,61 @@
+{ config, lib, pkgs, ... }:
+
+{
+  mobile.device.name = "motorola-potter";
+  mobile.device.identity = {
+    name = "Moto G5 Plus";
+    manufacturer = "Motorola";
+  };
+
+  mobile.hardware = {
+    soc = "qualcomm-msm8953";
+    ram = 1024 * 3;
+    screen = {
+      width = 1080; height = 1920;
+    };
+  };
+
+  mobile.boot.stage-1 = {
+    kernel.package = pkgs.callPackage ./kernel { };
+  };
+
+  mobile.device.firmware = pkgs.callPackage ./firmware {};
+
+  # WiFi needs firmware from the device. Refer to README.adoc in this
+  # directory for instructions on how to get it, then uncomment this
+  # and change the path appropriately
+
+  # hardware.firmware = [
+  #   (config.mobile.device.firmware.override {
+  #     modem = ../../vendor/modem;
+  #   })
+  # ];
+
```

Comment:
The end-user shouldn't need to edit this file.

This should be in the `README.adoc` file. Though to be fair generic firmware hints should also exist independent from all devices.


```suggestion
```

Reply:
For clarity: the user should be putting these settings in their configuration.nix instead of here?

Reply:
Yeah, in their system configuration (e.g. `/etc/nixos/configuration.nix`) or if they're building outside of the system, they can add it to the system configuration, (e.g. in `local.nix`).